README.md

```diff
@@ -225,6 +225,19 @@ If building code using the `System` global in Webpack, the following config is n
 }
 ```
 
+As of webpack@5.0.0-beta.15, you can access [SystemJS context](/docs/system-register.md#format-definition) via `__system_context__`:
+
+```js
+// Log the URL of the current SystemJS module
+// http://localhost:8080/dist/js/main.js
+console.log(__system_context__.meta.url);
+
+// Import a SystemJS module, with the current SystemJS module's url as the parentUrl
+__system_context__.import('./other-file.js').then(m => {
```

Comment:
Why not use `System.import`?

Reply:
I was under the impression that using system.register context’s import function would change the parentUrl, allowing for imports relative to the current module. Is that not the case?